### PR TITLE
Create Executive vote - November 14, 2024.md

### PR DESCRIPTION
**Pull request by 0xldr**

### CODE REVIEW

File: governance/votes/Executive vote - November 14, 2024.md
Comment (blimpa) [inline governance/votes/Executive vote - November 14, 2024.md:119]:
```suggestion
```

File: governance/votes/Executive vote - November 14, 2024.md
Comment (blimpa) [inline governance/votes/Executive vote - November 14, 2024.md:133]:
```suggestion
```

File: governance/votes/Executive vote - November 14, 2024.md
Comment (blimpa) [inline governance/votes/Executive vote - November 14, 2024.md:144]:
```suggestion
- **Authorization**: [Poll 1152](https://vote.makerdao.com/polling/QmQizL1F), [Poll 1153](https://vote.makerdao.com/polling/Qmbohkr5), [Poll 1155]https://vote.makerdao.com/polling/QmYqM8Yf), [Poll 1156](https://vote.makerdao.com/polling/QmXsXzot), [Poll 1157](https://vote.makerdao.com/polling/Qmf955yA)```

File: governance/votes/Executive vote - November 14, 2024.md
Comment (blimpa) [inline governance/votes/Executive vote - November 14, 2024.md:218]:
```suggestion
Community debate on these topics can be found on the Sky [Governance forum](https://forum.sky.money/). Please review any linked threads to inform your position before voting.
```

File: governance/votes/Executive vote - November 14, 2024.md
Comment (blimpa) [reply governance/votes/Executive vote - November 14, 2024.md:63]: I kind of feel this part is straddling somewhat awkwardly between being technical and not. May be just me

**Review by blimpa — CHANGES_REQUESTED** [review]
Nothing blockworthy, in my assessment  :+1:

File: governance/votes/Executive vote - November 14, 2024.md
Comment (0xldr) [reply governance/votes/Executive vote - November 14, 2024.md:63]: Yes, it's a bit difficult because we need to explain why a `drip` call will be in the code.

File: governance/votes/Executive vote - November 14, 2024.md
Comment (CivicSage3) [inline governance/votes/Executive vote - November 14, 2024.md:10]:
```suggestion
The Governance Facilitators, Sidestream and Dewiz have placed an executive proposal into the voting system. MKR/SKY Holders should vote for this proposal if they support the following alterations to the Sky Protocol.
```

File: governance/votes/Executive vote - November 14, 2024.md
Comment (CivicSage3) [inline governance/votes/Executive vote - November 14, 2024.md:26]:
```suggestion
- A Spark Proxy Spell at [0x8a3aaeAC45Cf3D76Cf82b0e4C63cCfa8c72BDCa7](https://etherscan.io/address/0x8a3aaeAC45Cf3D76Cf82b0e4C63cCfa8c72BDCa7) will be executed.
```

File: governance/votes/Executive vote - November 14, 2024.md
Comment (CivicSage3) [inline governance/votes/Executive vote - November 14, 2024.md:80]:
```suggestion
Note that DssVestSuckable for USDS and DSSVestTransferrable for SKY are not yet available. As a result, short streams for equivalent values of DAI and MKR are used. It is hoped that by the end of January, when these streams expire, it will be possible to switch the streams for these budgets to USDS and SKY, as detailed in The Atlas.
```
Agreed with blimpa

File: governance/votes/Executive vote - November 14, 2024.md
Comment (CivicSage3) [inline governance/votes/Executive vote - November 14, 2024.md:140]:
```suggestion
If this executive proposal passes, then the Spark Proxy Spell prepared by Phoenix Labs and reviewed by Wintermute and Certora at [0x8a3aaeAC45Cf3D76Cf82b0e4C63cCfa8c72BDCa7](https://etherscan.io/address/0x8a3aaeAC45Cf3D76Cf82b0e4C63cCfa8c72BDCa7) will be executed. The expected inclusions are:
```

**Comment by CivicSage3** [comment]
I've made the first run through the document as well and left some suggestions. I'll check it again in a few hours. On the other hand, we might need to wait until tomorrow with final review, if Rema decides to provide the rate changes for this spell.

**Comment by 0xldr** [comment]
# Executive Creation

## Creation

- [ ] Sync Community Git
- [ ] Ensure you are working from the latest master branch
- [ ] Read Associated [Executive Contents sheet](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit?pli=1#gid=1593813984)
- [ ] Copy the executive template
- [ ] Update filename
- [ ] Update dates - exact time is not required and should be left at midnight.

- [ ] Ensure the title refers to the most impactful/important actions included in the proposal.
- [ ] Ensure that the summary refers to all actions included in the proposal.

- [ ] Ensure that the publisher (entity publishing the executive on the voting portal) of the executive is listed in the preamble (at the entity level).
- [ ] Ensure that the writers of the executive code are listed in the preamble (at the entity level).
- [ ] Ensure that the reviewers of the executive code are listed in the preamble (at the entity level).

- [ ] Ensure that the office-hours modifier is mentioned as part of the executive copy, if present in the executive spell code.
- [ ] Ensure that it is clearly stated which items are affected by the GSM Delay.
- [ ] Ensure that the GSM Delay period is clearly communicated.
- [ ] Ensure that the proposal expiry is explicitly noted, including the length of time for which the proposal is valid.

## Items

- [ ] Add each item to the proposal details section
- [ ] Order the items by judged impact/importance. If different from the executive contents, reorder the executive contents sheet so that they match.

- [ ] For each item
	- [ ] Ensure the executive summary contains a high-level overview of every action contained in the proposal.
	- [ ] Ensure that each item or set of items in the executive proposal has a corresponding entry in the proposal details section.
	- [ ] Ensure that each item in the copy contains a link to the reasoning for its inclusion.
	- [ ] Ensure that each item in the copy contains a link to the authorization for its inclusion.
	- [ ] Ensure that each item in the copy contains any links required for voters to determine the context surrounding its inclusion.
	- [ ] Ensure that the framing and tense used for each item is future-conditional. Ie: "IF this executive proposal passes, THEN this action will take place."
	- [ ] Ensure that hyperlinks to documentation and/or context are added to key parameters, terms, and contracts where reasonable and possible.
	- [ ] Ensure that Ethereum and IPFS addresses are hyperlinked such that it is easy to view their details.
	- [ ] Ensure that parties implicated in the proposal outcome related to this item are aware of the wording used.

## Self-Review

- [ ] Do any '$' parameter signifiers remain? (spell address is unknown at this point)
- [ ] Do the executive title, summary, and executive summary accurately reflect executive copy content?
- [ ] Read the executive copy in full, with the above checklists and general readability in mind. Make any required changes.
- [ ] Run the copy through a spelling and grammar checker - do not blindly accept suggested changes.
- [ ] Inform the reviewer that the executive copy is ready to review
- [ ] Commit changes, and create a PR to the Community GitHub repository.

## Submission

- [ ] Merge approved executive copy PR.
- [ ] Inform technical teams that the copy is ready and provide the link.
- [ ] Once technical teams have provided a spell address, add it to the executive copy.
- [ ] Update the proposals.json to set the active proposal on the portal.
- [ ] Refresh portal cache (DUX/Jetstream can provide password)
- [ ] Inform the community of the new executive proposal.

File: governance/votes/Executive vote - November 14, 2024.md
Comment (CivicSage3) [inline governance/votes/Executive vote - November 14, 2024.md:79]:
```suggestion
Note that DssVestSuckable for USDS and DSSVestTransferrable for SKY are not yet available. As a result, short streams for equivalent values of DAI and MKR are used. It is hoped that by the end of January, when these streams expire, it will be possible to switch the streams for these budgets to USDS and SKY, as detailed in the Atlas.
```

**Comment by CivicSage3** [comment]
Just one typo corrected. The rest is good to go!

# Executive Document Review

## Content
- [x] Check the title refers to the most impactful/important actions included in the proposal.
- [x] Check that the summary refers to all actions included in the proposal.
- [x] Check that the publisher (entity publishing the executive on the voting portal) of the executive is listed in the preamble (at the entity level).
- [x] Check that the writers of the executive code are listed in the preamble (at the entity level).
- [x] Check that the reviewers of the executive code are listed in the preamble (at the entity level).
- [x] Check that the office-hours modifier is mentioned as part of the executive copy, if present in the executive spell code.
- [x] Check that it is clearly stated which items are affected by the GSM Delay.
- [x] Check that the GSM Delay period is communicated.
- [x] Check that the proposal expiry is explicitly noted, including the length of time for which the proposal is valid.

## Items

- [x] Check each item is present in the proposal details section
- [x] Check items are ordered by judged impact/importance

- [x] For each item
	- [x] Check the executive summary contains a high-level overview of the item.
	- [x] Check that each item or set of items in the executive proposal has a corresponding entry in the proposal details section.
	- [x] Check that the copy contains a link to the reasoning for its inclusion.
	- [x] Check that the copy contains a link to the authorization for its inclusion.
	- [x] Check that the copy contains any links required for voters to determine the context surrounding its inclusion.
	- [x] Check that the framing and tense used for each item is future-conditional. Ie: "IF this executive proposal passes, THEN this action will take place."
	- [x] Check that hyperlinks to documentation and/or context are added to key parameters, terms, and contracts where reasonable and possible.
	- [x] Check that Ethereum and IPFS addresses are hyperlinked such that it is easy to view their details.
	- [x] Check that parties implicated in the proposal outcome related to this item are aware of the wording used.

## Final Checks

- [x] Read the executive copy in full, with the above checklists and general readability in mind. Raise any issues with the author.

File: governance/votes/Executive vote - November 14, 2024.md
Comment (MakerJanSky) [reply governance/votes/Executive vote - November 14, 2024.md:79]: We used `The Atlas` in the poll today as it had been used in past copy. I don't have a strong opinion about either direction, but we should probably pick one for styling and stick to it.

File: governance/votes/Executive vote - November 14, 2024.md
Comment (MakerJanSky) [inline governance/votes/Executive vote - November 14, 2024.md:45]:
```suggestion
If this executive proposal passes, then the DIRECT-SPARK-DAI [buffer (`plan`)](https://sky-atlas.powerhouse.io/A.3.8.1.3.1.4_Buffer_Definition/039d633f-bccf-492f-9c6d-c167e8b5a292|57eaf45219be9dbc3712) parameter on mainnet will be increased by 50 million DAI from 50 million DAI to **100 million DAI**.
```

**Review by MakerJanSky — CHANGES_REQUESTED** [review]
Apologies, this stems from my mistake on the coordination sheet.

File: governance/votes/Executive vote - November 14, 2024.md
Comment (CivicSage3) [reply governance/votes/Executive vote - November 14, 2024.md:79]: I don't mind either way, but if we use "The", we need to change it in other places where Atlas is mentioned within this Executive copy. Right now, every mention is with "the".

Comment (0xldr) [reply governance/votes/Executive vote - November 14, 2024.md:79]: >every mention is with "the".

There was only one mention using "the" which has been changed in https://github.com/makerdao/community/pull/2786/commits/373c79a2d11f98e8763eabcb5764d6014d50038d

File: governance/votes/Executive vote - November 14, 2024.md
Comment (0xldr) [reply governance/votes/Executive vote - November 14, 2024.md]: fixed in https://github.com/makerdao/community/pull/2786/commits/0434f8718e4b63cfeb4bcc0f87b8a34ec3c8ea36

**Comment by 0xldr** [comment]
```
cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/0434f8718e4b63cfeb4bcc0f87b8a34ec3c8ea36/governance/votes/Executive%20vote%20-%20November%2014%2C%202024.md -O -)"
[0] Downloading 'https://raw.githubusercontent.com/makerdao/community/0434f8718e4b63cfeb4bcc0f87b8a34ec3c8ea36/governance/votes/Executive%20vote%20-%20November%2014%2C%202024.md' ...
HTTP response 200  [https://raw.githubusercontent.com/makerdao/community/0434f8718e4b63cfeb4bcc0f87b8a34ec3c8ea36/governance/votes/Executive%20vote%20-%20November%2014%2C%202024.md]
0xfb975e3cd0a4f127e0c19c95a778d5f3d4263de733a347f5cd7e7030252cb007
```

`0xfb975e3cd0a4f127e0c19c95a778d5f3d4263de733a347f5cd7e7030252cb007` matches the output of https://emn178.github.io/online-tools/keccak_256.html

File: governance/votes/Executive vote - November 14, 2024.md
Comment (EndgameEdge) [inline governance/votes/Executive vote - November 14, 2024.md:134]:
```suggestion
- **Authorization**: [Poll 1152](https://vote.makerdao.com/polling/QmQizL1F), [Poll 1153](https://vote.makerdao.com/polling/Qmbohkr5), [Poll 1155](https://vote.makerdao.com/polling/QmYqM8Yf), [Poll 1156](https://vote.makerdao.com/polling/QmXsXzot), [Poll 1157](https://vote.makerdao.com/polling/Qmf955yA)
```

File: governance/votes/Executive vote - November 14, 2024.md
Comment (EndgameEdge) [inline governance/votes/Executive vote - November 14, 2024.md:195]:
```suggestion
If this executive proposal passes the following change will be made to the SparkLend ETH Market [Interest Rate Model](https://sky-atlas.powerhouse.io/#A.3.8.1.6.1.3_Interest_Rate_Model_Definition-83e3f4f2-5e8e-4383-95e5-3af1bfb19210|57eaf45219be608847d6) on mainnet:
```

File: governance/votes/Executive vote - November 14, 2024.md
Comment (EndgameEdge) [inline governance/votes/Executive vote - November 14, 2024.md:201]:
```suggestion
If this executive proposal passes the following parameter changes will be made to the SparkLend wstETH borrow rate limits on mainnet:
```

**Review by VoteWizard — APPROVED** [review]
Great work, everyone!

**Comment by VoteWizard** [comment]
### Hash of the Document Before Merging

Command:
`cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/15c84e55529a280e29eeda56ec4bc0651ef42542/governance/votes/Executive%20vote%20-%20November%2014%2C%202024.md' -q -O - 2>/dev/null)"`

Hash:
`0x1bb8e29f12f8d0a4d03a82f8b9ad78945d3de288a79895142d213ceb2a6d0820`

The hash matches with the outcome of this online tool:
https://emn178.github.io/online-tools/keccak_256.html

**Pull request merged**
